main.cpp

```diff
@@ -1,8 +1,70 @@
-#include <iostream>
+/*#include <iostream>
 
 int main() {
     std::cout << "Run 'doxygen Doxyfile.in' from the root project directory to generate documentation." << std::endl;
 	std::cout << "View the documentation by opening docs/html/index.html in a web browser." << std::endl;
 	return 0;
+}*/
+
+#include <iostream>
```

Comment:
I think this code was used for testing but we don't want to modify the main.cpp file

Reply:
the test code should not have been checked in